W-21210749 split model policies gr 2 (#994)
mulesoft/docs-gateway: pdk/1.7/modules/ROOT/pages/policies-pdk-troubleshooting.adoc

To solve this error, refer to xref:policies-pdk-prerequisites.adoc#rust-requirements[Rust Requirements for Using PDK] to ensure your Rust software meets the required version.

[[implementation-build-no-file]]
== Error: No such file or directory When Running Make Build (Split Model Policy)

> When running `make build` from the implementation project, the following error might occur:

Reviewer: ```suggestion
When running `make build` from the implementation project, this error sometimes occurs:
```

[source,ssh]
----
Error: No such file or directory (os error 2)
make: *** [build] Error 1
----

> Ensure that you have executed `make setup`, `make release-local`, and published the definition asset version before building the implementation.

Reviewer: ```suggestion
Make sure that you've executed `make setup` and `make release-local` and that you've published the definition asset version before building the implementation.
```

[[build-asset-files-asset-not-found]]
== Error Looking for the Specified Asset When Running Make Build-Asset-Files (Split Model Policy)

> When running `make build-asset-files` from the implementation project, the following error might occur:

Reviewer: ```suggestion
When running `make build-asset-files` from the implementation project, this error sometimes occurs:
```

[source,ssh]
----
Fetching policy: {definition_asset_id}:{definition_asset_version}... !
    Error: There was a problem looking for the specified asset.
make: *** [build-asset-files] Error 2
----

> Ensure that you have executed `make setup`, `make release-local`, and published the definition asset version before building the implementation.

Reviewer: ```suggestion
Make sure that you've executed `make setup` and `make release-local` and that you've published the definition asset version before building the implementation.
```
